engine/include/cubos/engine/assets/bridges/json.hpp

```diff
@@ -60,21 +54,16 @@ namespace cubos::engine
         bool saveToFile(const Assets& assets, const AnyAsset& handle, core::memory::Stream& stream) override
         {
             // Initialize a JSON serializer with the file stream.
-            core::data::old::JSONSerializer serializer{stream, mIndentation};
+            core::data::JSONSerializer serializer{};
 
             // Read the asset from the asset manager and serialize it to the file stream.
             auto data = assets.read<T>(handle);
-            serializer.write(*data, nullptr); // JSON serializer does not use names.
-            if (serializer.failed())
-            {
-                CUBOS_ERROR("Could not serialize asset to JSON file");
-                return false;
-            }
+            serializer.write(*data);
 
+            // new JSONSerializer() does not receive a stream to write to, need to write to it manually
```

Comment:
You can remove those comments: future readers won't know about the old serialization system :relieved:

Reply:
You forgot to remove it!